homeassistant/components/buienradar/config_flow.py

```diff
@@ -23,6 +29,34 @@
     SUPPORTED_COUNTRY_CODES,
 )
 
+OPTIONS_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_COUNTRY, default=DEFAULT_COUNTRY): vol.In(
+            SUPPORTED_COUNTRY_CODES
+        ),
```

Comment:
I'm not sure this is correct. Isn't it the suggested_value that should fall back to `DEFAULT_COUNTRY`?
Same comment for the other options.